### ISSUE TITLE
Crash (ver 1.2.9) and doesn't remove all sponsor posts (ver 1.2.8)

### ISSUE THREAD

**Issue by NguyenASang** [jacobcxdev/FBSpNOsor#7]
_**About my device**_: My phone is iPhone 6, iOS 12.5.5, jailbreak with Chimera.
**_Problem_**: 
- Version 1.2.9: crashes my Setting when opening Setting panel of this tweak 
- Version 1.2.8: No crash but doesn't remove all sponsor post on my Facebook (latest version)

_**My opinion**_: 
- I guess the culprit of crash is with the things you changed in version 1.2.9 (link the changed things if you need: https://github.com/jacobcxdev/FBSpNOsor/compare/v1.2.8...v1.2.9)

_**In summary**_: I hope you can fix this soon, thank you for your great work and have a nice day.
